junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTestNameFormatter.java

```diff
@@ -91,10 +94,17 @@ private String argumentsPattern(Object[] arguments) {
 		Object[] result = Arrays.copyOf(arguments, Math.min(arguments.length, formats.length), Object[].class);
 		for (int i = 0; i < result.length; i++) {
 			if (formats[i] == null) {
-				result[i] = StringUtils.nullSafeToString(arguments[i]);
+				result[i] = truncateIfExceedsMaxLength(StringUtils.nullSafeToString(arguments[i]));
 			}
 		}
 		return result;
 	}
 
+	private String truncateIfExceedsMaxLength(String argument) {
+		if (argument.length() > argumentMaxLength) {
+			return argument.substring(0, argumentMaxLength - 1) + "\u2026";
```

Comment:
Maybe add a comment that the unicode character is an ellipsis.

Reply:
Done in 4f4707e082ae70e3f34925f0d1f37039b5d70691.